src/main/webapp/style.css

```diff
@@ -0,0 +1,28 @@
+#content {
+  margin: auto;
+  max-width: 100%;
+}
+
+div.header {
```

Comment:
Can you sync this with Sydney's latest when it's merged? Or you can just leave it out of the PR if you'd like. I think her changes will make at least some of this obsolete.